.azure/wheels.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install cibuildwheel==2.11.2
           pip install -U twine
+          rustup override set stable
```

Comment:
Is there a way to force a wheel build? To reduce the risk of breaking our release process.

Reply:
In pip it's
```
pip install --only-binary <comma,separated,packages> packageA packageB
```
where the comma-separated ones are the ones that are limited to be wheel-only, and `packageA`, `packageB` are the top level ones - the idea being that you can limit only a couple of the packages in the dependency stack if you want.  If you want to enforce it for all, then you can use `:all:` instead of the comma-separated list.  I _think_ at least a couple of our dependencies might not have wheels for all platforms, but the Azure ones should be fine.

Reply:
Ah, to clarify, I mean "is there a way to force the wheel Azure jobs to run"? That is, to do a dry-run of our deploy. 

https://github.com/Qiskit/qiskit-terra/blob/525f4dfe37a5934ad064a0b49b89b4145198a925/azure-pipelines.yml#L205-L207

Reply:
Ohhh sorry, I totally misunderstood.  We don't have a built-in dry-run method right now; in the past we've just done very ad-hoc stuff where we push a commit to a branch that has that conditional trigger (and the `twine upload` stuff) removed.  Having a proper way to do that would absolutely be helpful.

Reply:
https://github.com/Qiskit/qiskit-terra/issues/9594

In the meantime, I think we have to gamble this is okay. The syntax is valid, and we have Rustup available in our other Azure jobs, so I think this should be safe.